src/converters/toRpcHttp.ts

```diff
@@ -3,11 +3,13 @@
 
 import { RpcHttpData, RpcTypedData } from '@azure/functions-core';
 import { AzFuncSystemError } from '../errors';
+import { sendResponse } from '../http/httpProxy';
 import { HttpResponse } from '../http/HttpResponse';
+import { isHttpStreamEnabled } from '../setup';
 import { toRpcHttpCookie } from './toRpcHttpCookie';
 import { toRpcTypedData } from './toRpcTypedData';
 
-export async function toRpcHttp(data: unknown): Promise<RpcTypedData | null | undefined> {
+export async function toRpcHttp(invocId: string, data: unknown): Promise<RpcTypedData | null | undefined> {
```

Comment:
Nit: same suggestion as below. I think using the full name for `invocationId` is clearer and more consistent.